elasticdl_preprocessing/utils/analyzer_utils.py

```diff
@@ -0,0 +1,168 @@
+"""This util gets statistics from the environment and return a default value
+if there is no statistics in the environment. So user can set a default value
+to perform unit tests.
+"""
+import os
+
+import tensorflow as tf
+
+
+def get_min(feature_name, default_value):
+    """Get the min value of numeric feature from the environment.
+    Return the default value if there is no the statistics in
+    the environment.
+
+    Args:
+        feature_name: String, feature name or column name in a table
+        default_value: Float.
+
+    Return:
+        Float
+    """
+    env_name = feature_name + "_min"
```

Comment:
We can name it feature_name + "__min", because the feature name may contain '_', so we use '__' to separate feature name and analyze result key.

Reply:
Can you give an example to show the difference between "_" and ""?

Reply:
We can use `"_" + feature_name + "_min"` to separate them.